src/assert.js

```diff
@@ -0,0 +1,47 @@
+"use strict";
+
+/**
+ *  Assert checks the parameters.
+ */
+
+class Assert {
+
+    /**
+     *  Constructor.  
+     */
+    constructor() {
```

Comment:
You do not need constructor if it empty, it should be implicitly added in that case.